py_ts_interfaces/parser.py

```diff
@@ -74,11 +74,12 @@ def parse(self, code: str) -> None:
             self.prepared[current.name] = get_types_from_classdef(current)
         ensure_possible_interface_references_valid(self.prepared)
 
-    def flush(self) -> str:
+    def flush(self, should_export: bool) -> str:
```

Comment:
This change will likely cause a number of unit tests to fail that call this exact function without the new parameter